### PR TITLE
Added mobile responsiveness for Paragraphs w/signoff

### LINKED ISSUE #173: Padding and Text align in getnighthawk mobile view
**Description**

For the Mobile View,
1. The paragraph in the "Why Getnighthawk" section is end to end on the sides.
2. The paragraph in the "Easing Management of the Nighthawk Lifecyle" is left aligned.

**Expected Behavior**

1. Add some padding in the "Why Getnighthawk" paragraph section.
2. Change the text align to center in the "Easing Management of the Nighthawk Lifecycle" paragraph section and add some padding there also 

**Screenshots**

Screenshot_20210829-150830_Chrome
Screenshot_20210829-150843_Chrome

**Environment:**

- OS: [e.g. Ubuntu]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]
- Device: [e.g. laptop, iPhone 8]

---

[Optional] **To Reproduce**

[Optional] **Additional Context**

---

**Contributor Resources**

- See the [Contributing to getnighthawk.dev Website](https://github.com/layer5io/getnighthawk/blob/master/CONTRIBUTING.md)
- See the Project Site section of [GetNighthawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w)

### PR DESCRIPTION
**Pull request by AnshumanDhiman**
Signed-off-by: AnshumanDhiman <67755381+AnshumanDhiman@users.noreply.github.com>

Added padding and center aligned text for mobile viewers

This PR fixes #173 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/_sass/getnighthawk.scss [hunk 1]
```diff
@@ -6226,6 +6226,14 @@ margin: 0;
   .des-para {
     font-size: 14px;
   }
+  .des-para1 {
```
Comment (warunicorn19) [inline docs/_sass/getnighthawk.scss:6229]: What is this being used for🤔? don't see this class called anywhere

**Review by warunicorn19 — APPROVED** [review]
Changes look though, thanks @AnshumanDhiman

File: docs/_sass/getnighthawk.scss (hunk 1 above)
Comment (vinayaksh42) [reply docs/_sass/getnighthawk.scss:6229]: https://github.com/layer5io/getnighthawk/blob/master/docs/index.html#L192 looks like it is being used at this line @warunicorn19

**Review by vinayaksh42 — APPROVED** [review]
@AnshumanDhiman great work

File: docs/_sass/getnighthawk.scss (hunk 1 above)
Comment (warunicorn19) [reply docs/_sass/getnighthawk.scss:6229]: Ohhh okay😅

**Comment by AnshumanDhiman** [comment]
thanks @warunicorn19 @vinayaksh42 😊

**Pull request merged**